pkg/util/iptables/iptables.go

```diff
@@ -639,7 +645,7 @@ func getIPTablesVersion(exec utilexec.Interface, protocol Protocol) (*utilversio
 func getIPTablesWaitFlag(version *utilversion.Version) []string {
 	switch {
 	case version.AtLeast(WaitSecondsMinVersion):
-		return []string{WaitString, WaitSecondsValue}
+		return []string{WaitString, WaitSecondsValue, WaitIntervalString, WaitIntervalUsecondsValue}
```

Comment:
It looks like `-W` wasn't added until 1.6.1 so you'll need another case here.
(For iptables-restore both flags were added at the same time, in 1.6.2, so no change is needed there.)

Reply:
done